sdk/formrecognizer/ai-form-recognizer/tests.yml

```diff
@@ -1,21 +1,11 @@
-# Parameterize Location to the test resources deployment script. This allows
-# specifying a different Azure region to use when manually triggering the live
-# tests pipeline. This is useful for when we want to test in different
-# environments: Prod, Canary, etc.
-parameters:
-- name: Location
-  displayName: Location
-  type: string
-  default: canadacentral
-
 trigger: none
 
 stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       PackageName: "@azure/ai-form-recognizer"
       ServiceDirectory: formrecognizer
-      Location: "${{ parameters.Location }}"
+      SupportedClouds: 'Public,UsGov,China'
       EnvVars:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
```

Comment:
These will need to be changed to not be hardcoded to the public cloud values, but rather the env vars dynamically set by the test infra, e.g.

`AZURE_CLIENT_ID: $(FORMRECOGNIZER_CLIENT_ID)`

Reply:
But app-config has the same thing, why does it work over there?
![image](https://user-images.githubusercontent.com/10452642/156052610-58c87a93-8db2-4168-ba73-f31de814e3c2.png)